vpr/src/base/clustered_netlist.cpp

```diff
@@ -274,6 +276,28 @@ void ClusteredNetlist::shrink_to_fit_impl() {
     net_is_global_.shrink_to_fit();
 }
 
+/**
```

Comment:
Remove this data since it is only used in one parser and instead create locally (in a map or whatever you want there).  Easier to understand and maintain since the data is local and has a short lifetime then.
Right now you're adding global state that all netlist modifiers (delete_block for example) will have to keep up to date.  Since it's only used in one spot, it isn't worth it.